app/views/pages/profile.html.erb

```diff
@@ -36,9 +39,13 @@
         <p class="card-text"><%= @mentor_lesson.duration%></p>
         <p class="card-text"><%= @mentor_lesson.price%></p>
         <a href="<%= edit_lesson_path(@mentor_lesson) %>" class="btn btn-primary">Update lesson</a>
+
+
       </div>
     </div>
+    <% end %>
     <%# card mentor bookings  %>
+    <% if @mentor_bookings != nil %>
     <% if @mentor_bookings.count == 0 %>
       <h5>You have 0 bookings</h5>
```

Comment:
to be refactored

Reply:
REFACTORED:
<img width="1440" alt="Schermata 2020-10-28 alle 08 18 25" src="https://user-images.githubusercontent.com/70261924/97404318-32e7b700-18f6-11eb-9954-72d401bc0e41.png">
<% if @mentor_bookings.nil? %>
      <h5>You have 0 bookings</h5>
    <% else %>
      <h5>You have <%= @mentor_bookings.count %> bookings:</h5> <%# we need always more than 1 booking as seed %>
      <% @mentor_bookings.each do |booking| %>
        <%= booking.date %>
        <%= booking.user.name %> <%# student's name  %>
    <% end %>
  <% end %>